src/arch/arm/64/model/statedata.c

```diff
@@ -92,9 +92,9 @@ pte_t armKSGlobalKernelPT[BIT(PT_INDEX_BITS)] ALIGN_BSS(BIT(seL4_PageTableBits))
 #ifdef CONFIG_KERNEL_LOG_BUFFER
 pte_t *armKSGlobalLogPDE = &armKSGlobalKernelPDs[BIT(PT_INDEX_BITS) - 1][BIT(PT_INDEX_BITS) - 2];
 compile_assert(log_pude_is_correct_preallocated_pude,
-               GET_PUD_INDEX(KS_LOG_PPTR) == BIT(PT_INDEX_BITS) - 1);
+               GET_KPT_INDEX(KS_LOG_PPTR, 1) == BIT(PT_INDEX_BITS) - 1);
```

Comment:
Just wondering if this should be:
```suggestion
               GET_KPT_INDEX(KS_LOG_PPTR, KLVL_FRM_ARM_PT_LVL(1)) == BIT(PT_INDEX_BITS) - 1);
```

As per: https://github.com/seL4/seL4/blob/91ec17c5bf29a4ef8b98b8f9769580b2442eb810/src/arch/arm/64/kernel/vspace.c#L237